package.json

```diff
@@ -4,11 +4,9 @@
   "version": "2.5.1",
   "main": "run.js",
   "scripts": {
-    "lint": "./node_modules/.bin/eslint lib spec mocha_spec lib run.js runService.js",
```

Comment:
Please, keep pretest script and check errors

Reply:
Please see this PR needs to be merged first https://github.com/elasticio/sailor-nodejs/pull/119